tests/check/check_signed_video_auth.c

```diff
@@ -491,6 +491,12 @@ START_TEST(remove_one_p_nalu)
   // authenticity is NOT OK.
   signed_video_accumulated_validation_t final_validation = {
       SV_AUTH_RESULT_NOT_OK, false, 14, 13, 1, SV_PUBKEY_VALIDATION_NOT_FEASIBLE, true, 0, 0};
+  // SIPPSIPPSIPPSI
+  //
+  // SI                ->   (valid) -> .P
+  //  IPPSI            ->   (valid) -> .....P
+  //      IPPSI        -> (invalid) -> NNNNP
```

Comment:
I think you are misunderstanding the review comments. First of all, since we now only apply the Document principle, all SEIs where the signature can be verified successfully should be marked '.'. Secondly, since the SEI can be verified successfully it is possible to detect how many NALUs there are missing, BUT if there is no hash list it is not possible to know which hash(es) are missing. They will either be added right after the I-frame or after the last frame of the GOP. You can look in the debug prints which it is. So I would expect this pattern to be either `NMNN.P` or `NNNM.P`

Reply:
Currently, the SEI is marked as |used_in_gop_hash|, and the validation status is set within the GOP. I have already added a TODO comment in the 
|prepare_for_link_and_gop_hash_verification| function. I plan to change this behavior when removing the first SEI pull request.